docs/content/primary-key-table/merge-engine.md

```diff
@@ -247,6 +247,10 @@ Current supported aggregate functions and data types are:
   The first_non_null_value function selects the first non-null value in a data set.
   It supports all data types.
 
+* `theta_sketch`:
```

Comment:
can you show an example to how to use sketch to finish business?

Reply:
done.